packages/pyright-internal/src/parser/stringTokenUtils.ts

```diff
@@ -142,7 +146,7 @@ export function getUnescapedString(stringToken: StringToken | FStringMiddleToken
     while (true) {
         let curChar = getEscapedCharacter();
         if (curChar === Char.EndOfText) {
-            return completeUnescapedString(output);
+            return completeUnescapedString(output, escapedString);
```

Comment:
Found this one too. Netted another 200K in memory savings with the test app I'm using